### PR TITLE
Update carddata.txt

### PR DESCRIPTION
**Pull request by RrTe**
Adjusted the following parts beginning from here: https://www.cactusforums.com/redemption-gaming-on-lackeyccg/lackey-edits/msg595342/#msg595342

- Egyptian Wise Men (FoM) has wrong SpecialAbility text.
- Seized by Babylon has as Set "MaProphecies of Christ"
- Abram/Abraham (Patriarchs) has Strenght 10-May and Toughness 11-Jun
- Philosophy (Promo) has Strength 2-Mar and Toughness 3-Feb
- Family Bond is listed as only gold, when it should be gold/white.
- Outsiders AB (Gold/Black) is listed as an EE
- Outsiders AB (Crimson/Brown) is listed as an EE
- "When Judges Governend" from J deck is missing white brigade
- replaced several � with '
- Oath of Purity (LoC) has wrong rarity - should be common
- Remnant LS is misspelled as Remant.
- Abishai (Ki) is wrongly written and recognised as Abishai (Ki() in cardpool and deck files.
- Blindness (PoC) is typed as EE/Artifact instead of Curse.
- Alliance Against Judah is named as Alliance Against Israel

Following parts from that thread already have been corrected:
- Stephen (EC) is missing identifiers Deacon, Jerusalem
- Nicanor (EC) is missing identifiers Deacon, Jerusalem
- Roman Citizenship (PC) is missing red as brigade
- The Storehouse promo and False Dreams (PoC) is missing.
- Jeremiah, Hope Bringer is called Treacherous Land
- Ehud's Dagger (UL) Red shows wrong picture (Desecrate the Temple)

Following parts could not be solved:
- If old New Jerusalem Promo is chosen for deck then it is shown correctly regarding image within deck editor but not within game and hand and deck in game! There the picture of the new NJ Promo is shown. Maybe image names of both cards are equal?
- Just wanted to inquire if we could get Foolish Shepherd in Lackey now that Rob has the Mourn & Weep trade in program active for it? Basically now the card is available to people.

### CODE REVIEW

File: RedemptionQuick/sets/carddata.txt [hunk 1]
```diff
@@ -49,7 +49,7 @@ Abner's Spear (Wa)	Warriors	Abner's_Spear_(Wa)	EE	Brown	2	4				Common	II Samuel
...
 Abraham's Descendant	Disciples	Abraham's_Descendant_(Di)	GE	Blue	3	3	Territory	Prophesy	Interrupt a Curse or Evil Enhancement in play and add it to your hand.  Search each discard pile for a good Enhancement and place them beneath owners' decks.	Fixed	Genesis 12:3	
 Abraham's Servant to Ur	Patriarchs	Abraham's_Servant_to_Ur_(Pa)	GE	Blue	2	3			All special abilities except banding on characters and enhancements, except this one, are negated.  Battle is determined by the numbers.	Rare	Genesis 24:10	
-Abram/Abraham	Patriarchs	Abram_Abraham_(Pa)	Hero	Blue	10-May	11-Jun			Hero starts as Abram.  After he makes a successful rescue, he becomes Abraham and can interrupt and prevent all special abilities on all Lost Soul cards except the Proverbs 22:14 Lost Souls card.	Ultra Rare	Genesis 17:4-5	
+Abram/Abraham	Patriarchs	Abram_Abraham_(Pa)	Hero	Blue	10	11			Hero starts as Abram.  After he makes a successful rescue, he becomes Abraham and can interrupt and prevent all special abilities on all Lost Soul cards except the Proverbs 22:14 Lost Souls card.	Ultra Rare	Genesis 17:4-5
```
Comment (MattJBrinkman) [inline RedemptionQuick/sets/carddata.txt:52]: Abram/Abraham should have strength 5/10  and toughness 6/11

File: RedemptionQuick/sets/carddata.txt [hunk 2]
```diff
@@ -139,7 +139,7 @@ Angel of the Lord (I)	I	Angel_of_the_Lord_(I)	Dominant	Good					Discard an Evil
 Angel of the Lord (J)	J	Angel_of_the_Lord_(J)	Dominant	Good					Discard an Evil Character.	Deck	Exodus 3:2	
 Angel of the Waters	Kings	Angel_of_the_Waters_(Ki)	Hero	Silver	6	8			Silver brigade Heroes cannot be taken prisoner this turn.	Uncommon	Revelation 16:5	
 Angel of Warning (Ap)	Apostles	Angel_of_Warning_(Ap)	Hero	Silver	4	4				Common	Matthew 2:13	
-Angel of Warning (TxP)	Thesaurus ex Preteritus	Angel_of_Warning_(TxP)	Hero	Silver	4	4		Generic	May look at opponent�s hand, or search deck for a Site or a good Fortress and put it in play.	Fixed	Mattehw 2:13	
+Angel of Warning (TxP)	Thesaurus ex Preteritus	Angel_of_Warning_(TxP)	Hero	Silver	4	4		Generic	May look at opponent's hand, or search deck for a Site or a good Fortress and put it in play.	Fixed	Mattehw 2:13
```
Comment (MattJBrinkman) [inline RedemptionQuick/sets/carddata.txt:142]: Can you fix reference to be Matthew

File: RedemptionQuick/sets/carddata.txt [hunk 3]
```diff
@@ -803,10 +803,10 @@ False Teachers (L)	Main	False_Teachers_(UL)	Evil Character	Gray	6	6				Uncommon
 False Teachers (UL)	Main Unlimited	False_Teachers_(UL)	Evil Character	Gray	6	6				Uncommon	Jude 4	
 False Wisdom	Prophets	False_Wisdom_(Pr)	EE	Gray					Holder may look at rescuer's hand.	Common	Isaiah 47:10	
 False Witnesses	10th Anniversary	False_Witnesses_(G)	EE	Black	2	2			If used by a Sadducee, capture a hero.	Deck	Matthew 26:59	
-Family Bond	J	Family_Bond_(J)	GE	Gold	2	2			Negate and discard the last evil enhancement played this battle. 	Deck	Ruth 1:16	
+Family Bond	J	Family_Bond_(J)	GE	White/Gold	2	2			Negate and discard the last evil enhancement played this battle. 	Deck	Ruth 1:16
```
Comment (MattJBrinkman) [inline RedemptionQuick/sets/carddata.txt:806]: Should by Gold/White  for consistency

File: RedemptionQuick/sets/carddata.txt [hunk 4]
```diff
@@ -1823,7 +1823,7 @@ Philistine Chariot and Horses	Thesaurus ex Preteritus	Philistine_Chariot_and_Hor
 Philistine Garrison	Thesaurus ex Preteritus	Philistine_Garrison_(TxP)	Evil Character	Black	12	8	Warrior	Generic, Philistine, Fought Earthly battle	Immune to lone Heroes.	Fixed	I Samuel 14:12	
 Philistine Outpost	Thesaurus ex Preteritus	Philistine_Outpost_(TxP)	Fortress	Evil					When you are attacked, you may discard this card or an evil card from deck to search discard pile for a generic Philistine and place it in your territory. Protect Sites from being placed beneath decks.	Fixed	I Samuel 14:12	
 Philistine Priests	Priests	Philistine_Priests_(Pi)	Evil Character	Black	5	5		Generic Priest	You may search draw pile for Temple of Dagon and put it in play.  If another Philistine Evil Character is in play, you may negate and deactivate one O.T.  Artifact in play.	Common	I Samuel 6:2	
-Philosophy	Promo	Philosophy_(good)_(Promo)	DAE	Multi	2-Mar	3-Feb				Promo	I Corinthians 1:20,25 (Good); Proverbs 1:29, 32 (Evil)	
+Philosophy	Promo	Philosophy_(good)_(Promo)	DAE	Multi	3	2				Promo	I Corinthians 1:20,25 (Good); Proverbs 1:29, 32 (Evil)
```
Comment (MattJBrinkman) [inline RedemptionQuick/sets/carddata.txt:1826]: Philosophy should have Strength  3/2  and Toughness 2/3

File: RedemptionQuick/sets/carddata.txt (hunk 1 above)
Comment (MattJBrinkman) [inline RedemptionQuick/sets/carddata.txt:52]: Abram/Abraham should have strength 5/10 and toughness 6/11

File: RedemptionQuick/sets/carddata.txt (hunk 2 above)
Comment (MattJBrinkman) [inline RedemptionQuick/sets/carddata.txt:142]: Can you fix reference to be Matthew

File: RedemptionQuick/sets/carddata.txt (hunk 3 above)
Comment (RrTe) [reply RedemptionQuick/sets/carddata.txt:806]: Found that some of the cards had their 1st brigade written first, their 2nd afterwards etc. and for some other cards it was vice versa. Is there a pattern?

File: RedemptionQuick/sets/carddata.txt (hunk 4 above)
Comment (RrTe) [reply RedemptionQuick/sets/carddata.txt]: Corrected

File: RedemptionQuick/sets/carddata.txt (hunk 1 above)
Comment (RrTe) [reply RedemptionQuick/sets/carddata.txt]: Corrected

File: RedemptionQuick/sets/carddata.txt (hunk 2 above)
Comment (RrTe) [reply RedemptionQuick/sets/carddata.txt]: Fixed

File: RedemptionQuick/sets/carddata.txt (hunk 3 above)
Comment (MattJBrinkman) [reply RedemptionQuick/sets/carddata.txt:806]: Good then Evil
Alphabetical within good and evil.
It gives a consistent way for users to know how to search.

**Pull request merged**